### ISSUE TITLE
Mess in the manage course and category page

### ISSUE THREAD

**Issue by adamtong1110** [eduridden/moodle-theme_elegance#57]
This theme is not doing well in the following page:
course/management.php

After i click to expand the category, the course list shown below, (should be on the right in original way)

Thanks.

**Comment by thedannywahl** [comment]
I'm assuming you're talking about "Course Categories and courses" and NOT "Courses" or "Course Categories" correct? and of course this is 2.7 (dev).  The issue is that the Moodle core is using bootstrap 2, but this theme is bootstrap 3. A renderer may be possible.

'More' theme:
![screen shot 2014-04-09 at 2 47 59 pm](https://cloud.githubusercontent.com/assets/743499/2652650/eefcab70-bfb2-11e3-8eb7-639574fed986.png)

**Comment by thedannywahl** [comment]
https://github.com/moodle/moodle/blob/master/course/classes/management_renderer.php#L842-L964

this will need to be rewritten to be BS3 compatible.

**Comment by adamtong1110** [comment]
yes, i am talking exactly the same page as your print screen. How can I get this page as nice as your print screen in this theme?

**Comment by adamtong1110** [comment]
I just installed Moodle 2.7 for testing the above page, however, it is still the same as in moodle 2.6.

But if using core theme (Clean or More), it will show very nicely.

**Comment by thedannywahl** [comment]
@adamtong1110 go ahead and open an issue on my fork - I can't do anything with them here.  Thanks.

**Comment by adamtong1110** [comment]
open an issue to where??

**Comment by thedannywahl** [comment]
https://github.com/thedannywahl/moodle-theme_elegance/issues
